woocommerce/woocommerce-ios
# Add API request URLs to logs

Closes #1005.

This PR is meant to enhance the networking env. work for devs.

**My only concern is this:** does it introduce any security risks? I don't think it does, because I'm not logging headers, and I think every API we use it publicly available.

## To test
- Check out this PR
- Run the app, navigate around a bit
- Check the Settings > Help & Support > Application Logs for print outs of API requests

Update release notes:
